lib/internal/bootstrap_node.js

```diff
@@ -54,21 +49,11 @@
     NativeModule.require('internal/process/warning').setup();
     NativeModule.require('internal/process/next_tick').setup();
     NativeModule.require('internal/process/stdio').setup();
+
+    const browserGlobals = !process._noBrowserGlobals;
     if (browserGlobals) {
-      // Instantiate eagerly in case the first call is under stack overflow
-      // conditions where instantiation doesn't work.
```

Comment:
The previous commit here still seems valid...?

Reply:
Somewhat yes, but it did not fully work as anticipated and therefore I think it is fine to remove the comment. I can add the comment back in if you like though.

Reply:
There's 3

Reply:
DEA